conda/cli/conda_argparse.py

```diff
@@ -115,7 +120,9 @@ def do_call(arguments: argparse.Namespace, parser: ArgumentParser):
     Serves as the primary entry point for commands referred to in this file and for
     all registered plugin subcommands.
     """
-    # First, check if this is a plugin subcommand; if this attribute is present then it is
+    if hasattr(arguments, "no_plugins") and arguments.no_plugins or context._no_plugins:
```

Comment:
Shouldn't this just refer to `context.no_plugins`? I think it would be good to implement this logic in one place.